src/main/java/core/basesyntax/service/impl/operation/ReturnOperation.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.Fruit;
+
+public class ReturnOperation implements OperationHandler {
+    private final FruitStorageDao fruitStorageDao;
+
+    public ReturnOperation(FruitStorageDao fruitStorageDao) {
+        this.fruitStorageDao = fruitStorageDao;
+    }
+
+    @Override
+    public boolean doOperation(Fruit fruit, int quantity) {
+        if (fruit == null) {
+            throw new RuntimeException("Fruit can't be null");
+        }
+        if (quantity < MIN_FRUIT_QUANTITY) {
```

Comment:
It appears that `MIN_FRUIT_QUANTITY` is not defined in this class. Ensure that you have a constant `MIN_FRUIT_QUANTITY` defined, or if it's supposed to be zero, you might want to compare directly with zero for clarity.